inc/tag.class.php

```diff
@@ -723,14 +723,14 @@ function checkMandatoryFields($input = []) {
     */
    public static function getCurrentItemtype() {
       $itemtype = '';
-      if (preg_match('/\/plugins\/([a-zA-Z]+)\/front\/([a-zA-Z]+).form.php/',
+      if (preg_match('/\/(marketplace|plugins)\/([a-zA-Z]+)\/front\/([a-zA-Z]+).form.php/',
```

Comment:
I think you have to use a non-capturing group to not change matches indexes.
```suggestion
      if (preg_match('/\/(?:marketplace|plugins)\/([a-zA-Z]+)\/front\/([a-zA-Z]+).form.php/',
```